Frameworks/UIKit/UIButton.mm

```diff
@@ -891,22 +946,18 @@ - (UIImageView*)imageView {
 }
 
 /**
- @Status Caveat
- @Notes Returns the receiving view
-*/
-- (UIView*)viewForBaselineLayout {
-    return self;
-}
-
-/**
  @Status Interoperable
 */
 - (CGSize)intrinsicContentSize {
     CGSize ret = CGSizeZero;
 
     UIImage* img = self.currentImage;
     UIEdgeInsets contentInsets = self.contentEdgeInsets;
-    CGSize textSize = [[self currentTitle] sizeWithFont:self.font];
+
+    // TODO: Should we be asking our label for its intrinsicContentSize?
```

Comment:
and i guess we have not run into a button which should display two line of text yet. :)

---
In reply to: [100654429](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100654429) [](ancestors = 100654429,100653705,100653270,100653027)